easier68k/core/opcodes/Bcc.py

```diff
@@ -0,0 +1,737 @@
+
+
+class BGE(branch_code):
+    """
+    The BGE opcode compiles down to 6Cxx xxxx xxxx
+    The conditional here is N AND V OR NOT N AND NOT V
+    """
+
+    cond_code = '\x0C'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        return n == v
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BGE')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BGE, ord('\x0C'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BGE)
+
+
+class BLT(branch_code):
+    """
+    The BLT opcode compiles down to 6Dxx xxxx xxxx
+    The conditional here is N AND NOT V OR NOT N AND V
+    """
+
+    cond_code = '\x0D'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        return not n == v
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BLT')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BLT, ord('\x0D'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BLT)
+
+
+class BGT(branch_code):
+    """
+    The BGT opcode compiles down to 6Exx xxxx xxxx
+    The conditional here is N AND V AND NOT Z OR NOT N AND NOV AND NOT Z
+    """
+
+    cond_code = '\x0E'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+        z = simulator.get_condition_status_code(ConditionStatusCode.Z)
+
+        return (n == v) and not z
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BGT')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BGT, ord('\x0E'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BGT)
+
+
+class BLE(branch_code):
+    """
+    The BLE opcode compiles down to 6Fxx xxxx xxxx
+    The conditional here is (Z) OR (V AND NOT Z) OR (NOT N AND NOT V AND NOT Z)
+    """
+
+    cond_code = '\x0F'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+        z = simulator.get_condition_status_code(ConditionStatusCode.Z)
+
+        return (n != v) and not z
```

Comment:
The Z condition is not correct here.

```suggestion
        return (n != v) or z
```